pom.xml

```diff
@@ -545,12 +545,24 @@
                             <user>1000</user>
                         </container>
                         <extraDirectories>
-                            <paths>src/main/docker/jib</paths>
+                            <paths>
+                                <path>
+                                    <from>src/main/docker/jib</from>
+                                    <into>/</into>
+                                    <excludes>
+                                        <exclude>**/.gitkeep</exclude>
+                                    </excludes>
+                                </path>
+                            </paths>
                             <permissions>
                                 <permission>
                                     <file>/entrypoint.sh</file>
                                     <mode>755</mode>
                                 </permission>
+                                <permission>
+                                    <file>/target</file>
```

Comment:
any reason to include this?

Reply:
This directory is required because application.yml set ` logging.file.name` to  `target/jhipster-registry.log`. 

The Docker image generated with `./mvnw package verify jib:dockerBuild ` doesn't include his directory, so it is not possible to boot them. However, the image generated from the docker file includes it and it works (see [dockerfile ](https://github.com/jhipster/jhipster-registry/blob/main/Dockerfile#L25))

So I supposed that both commands should generate a valid docker image.